qemu/tests/cfg/blockdev_inc_backup_migrate_without_bitmap.cfg

```diff
@@ -50,3 +50,5 @@
         - non_persistent:
             bitmap_debugged = no
             full_backup_options = {"sync": "full", "persistent": "off"}
+    iscsi_direct:
```

Comment:
remove the two lines for we've already defined it, see line26

Reply:
Have updated it, thanks zhenchao.